pages/linux/vgscan.md

```diff
@@ -0,0 +1,13 @@
+# vgscan
+
+> Scan for volume groups on all supported Logical Volume Manager (LVM) block devices.
+> See also `lvm`, `vgchange`.
+> More information: <https://manned.org/vgscan>.
+
+- Scan for volume groups and print information about each group found:
+
+`sudo vgscan`
+
+- Scan for volume groups and add or remove the special files in `/dev` needed to access the logical volumes in a group:
+
+`sudo vgscan --mknodes`
```

Comment:
So does it add or remove files? Or does that depend on the conext? And since you said `in a group`, does that mean it only does this for one group? If yes, wouldn't you have to specify its name?

Reply:
It is supposed to do both depending on context. I've only verified the addition. How about if I change the example description to:

```suggestion
- Scan for volume groups and add, if they don't already exist, the special files in `/dev` needed to access the logical volumes in the found groups:

Reply:
Sounds better, however I'm not sure about the grammar here. @bl-ue?

Reply:
Yep that's perfectly good grammar, but it might be confusing to non-native speakers. How about moving `if they don't already exist` to the end, @aherst?
```suggestion
- Scan for volume groups and add the special files in `/dev` needed to access the logical volumes in the found groups, if they don't already exist:

`sudo vgscan --mknodes`
```

Reply:
Much better

Reply:
I want to make sure it's clear that the clause applies to "files" and not to "groups". How about:

```suggestion
- Scan for volume groups and add the special files in `/dev`, if they don't already exist, needed to access the logical volumes in the found groups :

Reply:
sgtm